puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pipelines/katello.groovy

```diff
@@ -36,8 +36,8 @@ pipeline {
             steps {
                 script {
                     runCicoJobsInParallel([
-                        ['name': 'centos7 install test', 'job': "foreman-katello-${katello_version}-test"],
-                        ['name': 'centos7 upgrade test', 'job': "foreman-katello-upgrade-${katello_version}-test"]
+                        ['name': 'centos7 install test', 'job': "foreman-katello-${katello_version}-install-test"],
```

Comment:
this should also run el8 install test I guess? but then how would this step handle the fact that el8 tests don't exist for some versions?

Reply:
Good find. I've made this dynamic now based on configuration.